### PR TITLE
Design Mode UI fixes

### PR DESCRIPTION
**Pull request by joshlory**

### CODE REVIEW

File: apps/src/applab/designElements/textarea.jsx [hunk 1]
```diff
@@ -134,6 +134,7 @@ module.exports = {
     element.style.height = '100px';
     element.style.fontSize = '14px';
     element.style.color = '#000000';
+    element.style.backgroundColor = '#ffffff';
     element.style.backgroundColor = '';
```
Comment (mehalshah) [inline apps/src/applab/designElements/textarea.jsx]: Why set this twice?

**Comment by davidsbailey** [comment]
In addition to images without src no longer having an outline, they will no longer occlude elements which they sit in front of. Is this intended?

**Comment by davidsbailey** [comment]
It seems like `outline: none;` might be better for images with no src. otherwise LGTM

File: apps/src/applab/designElements/textarea.jsx (hunk 1 above)
Comment (joshlory) [inline apps/src/applab/designElements/textarea.jsx]: Whoops, good catch.

**Comment by joshlory** [comment]
Setting `outline: none;` on images without a src has no effect (since the outline comes from the browser, not CSS).  I think it's by-design that images without a src don't show up at all except for design mode.

**Pull request merged**